### PR TITLE
Correctly handle dumps with empty configs

### PR DESCRIPTION
**Pull request by nazywam**

### CODE REVIEW

File: karton/config_extractor/config_extractor.py [hunk 1]
```diff
@@ -212,14 +211,20 @@ def analyze_sample(self, task: Task, sample: ResourceBase) -> None:
...
-        if configs:
-            config = configs[0]
-            self.log.info("Got config: {}".format(json.dumps(config)))
-            self.report_config(task, config, sample)
-        else:
-            self.log.info("Failed to get config")
+        for config in extractor.config:
+            legacy_config, karton_tasks = self.preprocess_config(config)
+
+            if len(legacy_config.items()) > 1:
```
Comment (msm-cert) [inline karton/config_extractor/config_extractor.py:218]:
```suggestion
            if len(legacy_config) > 1:
```

File: karton/config_extractor/config_extractor.py [hunk 2]
```diff
@@ -212,14 +211,20 @@ def analyze_sample(self, task: Task, sample: ResourceBase) -> None:
...
+                self.log.info("Got config: {}".format(json.dumps(config)))
```
Comment (msm-cert) [inline karton/config_extractor/config_extractor.py]:
```suggestion
                self.log.info("Got config: %s", json.dumps(config))
```

**Review by msm-cert — APPROVED** [review]
:ack:

File: karton/config_extractor/config_extractor.py [hunk 3]
```diff
@@ -273,23 +278,36 @@ def analyze_dumps(
...
-                    "tags": ["dump:win32:exe"],
-                },
-            )
-            self.send_task(task)
-            self.report_config(task, config, sample, parent=parent)
+
+            legacy_config, karton_tasks = self.preprocess_config(config)
+            karton_task_parent = sample
+
+            if len(legacy_config.items()) > 1:
```
Comment (nazywam) [inline karton/config_extractor/config_extractor.py:285]:
```suggestion
            if len(legacy_config) > 1:
```

**Pull request merged**